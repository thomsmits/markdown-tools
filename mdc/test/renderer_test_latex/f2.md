# Chapter
## Slide

A __Term__
