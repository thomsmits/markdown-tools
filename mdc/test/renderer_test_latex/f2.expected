# Chapter

## Slide

A \term{Term}\index{Term}\vspace{0.1mm}
